.env

```diff
@@ -0,0 +1,6 @@
+REACT_APP_API_KEY=AIzaSyA1w2FNcFGvyFjgh1em0ZchR5mQ5MNxSFE
+REACT_APP_AUTH_DOM=laa-dashboard-test.firebaseapp.com
+REACT_APP_DB_URL=https://laa-dashboard-test.firebaseio.com
+REACT_APP_MESSAGE_ID=779550087954
+REACT_APP_ADMIN_MESSAGE="Development Status Admin - using test database"
+REACT_APP_ADMIN_STYLE="w3-container w3-blue"
```

Comment:
I know this hasn't changed as part of the PR, so don't go changing this as part of this PR.

...but it's a bit weird having this as a config option.  I'd suggest `REACT_APP_ADMIN_THEME_COLOUR=blue`.

Reply:
Oh wait, you did introduce this as part of the PR.  🤦‍♂️

... and that's why you read the whole PR before making comments.

Reply:
> REACT_APP_ADMIN_STYLE="w3-container w3-blue"
Yeah, you got me, that was lazy, I'll fix.